src/plugins/robot-memory/robot_memory.cpp

```diff
@@ -877,9 +877,32 @@ RobotMemory::mutex_try_lock(const std::string& name,
 		        new_doc.getField("locked").Bool());
 
 	} catch (mongo::OperationException &e) {
-		//if (e.obj()["code"].numberInt() != 11000) {
-		// 11000: Duplicate key exception, occurs if we do not become leader, all fine
+		logger_->log_error(name_, "Mongo OperationException: %s", e.what());
+		try {
+		mongo::BSONObjBuilder check_doc;
```

Comment:
Indentation error, inner block in try needs to be indented.

Reply:
Ah yes, I tried to adapt the formatting to smart tabs, I missed that one.

Reply:
The problem still isn't resolved, the try-block still has the wrong indentation. The last commit only changed the catch block. The alignment of the additional log_info parameters (the message and name) is also wrong, it should align with the beginning of "name_" the line before.